app/models/user.rb

```diff
@@ -106,6 +117,14 @@ def valid_persistence_token?(token)
     persistence_token == token
   end
 
+  def local_roles
+    self.class.local_roles
+  end
+
+  def global_roles
+    self.class.global_roles
+  end
+
```

Comment:
This could be:

``` ruby
delegate :local_roles, :global_roles, to: :class
```

See the documentation for [`delegate`](https://api.rubyonrails.org/classes/Module.html#method-i-delegate)